cvmfs/cvmfs_config

```diff
@@ -707,7 +707,12 @@ cvmfs_chksetup() {
       done
 
       # Check for network
-      local uuid=$(uuidgen)
+      if which uuidgen 1>/dev/null 2>&1; then
```

Comment:
The `which` utility comes in another package. The check can be done without the dependency on `which` simply by calling `uuidgen -V`.

Reply:
Thanks. Should be fine now.